builders/xcm/xcm-sdk.md

```diff
@@ -0,0 +1,678 @@
+---
+title: XCM SDK
+description: Use the Moonbeam XCM SDK to easily deposit and withdraw cross chain assets to Moonbeam from Polkadot and other parachains in the ecosystem.
+---
+
+# Using the Moonbeam XCM SDK
+
+![XCM SDK Banner](/images/builders/xcm/sdk/xcm-sdk-banner.png)
+
+## Introduction {: #introduction }
+
+The Moonbeam XCM SDK enables developers to easily deposit and withdraw assets to Moonbeam/Moonriver from the relay chain and other parachains in the Polkadot/Kusama ecosystem. With the SDK, you don't need to worry about determining the multilocation of the origin or destination assets or which extrinsics are used on which networks to send XCM transfers. To deposit or withdraw assets, you simply define the asset and origin chain you want to deposit from or withdraw back to, along with the sending account's signer, and the amount to send.
+
+The examples in this guide are shown on Moonbeam, but can be adapted to be used on Moonriver or Moonbase Alpha.
+
+Please note that the SDK is currently in heavy development and is not production ready.
+
+## Getting Started {: #getting-started }
+
+To get started with the XCM SDK, you'll first need to install the corresponding NPM package. You'll also need to create signers to be used to sign transactions to transfer assets between Moonbeam and another chain within the Polkadot ecosystem. Lastly, you'll need to initialize the API, which will provide you with asset and chain information and the necessary functions to deposit, withdraw, and subscribe to balance information.
+
+### Installation {: #installation }
+
+You'll need to install two packages for the purposes of this guide, the first being the XCM SDK package, and the second being the XCM config package. The XCM SDK package will enable you to easily deposit and withdraw assets, and subscribe to balance information for each of the supported assets. The XCM config package will be used to obtain origin asset and chain information for each of the supported assets. The config package also includes native asset and chain information for each of the Moonbeam-based networks, as well as some underlying functions of the SDK.
+
+To install the XCM SDK and XCM config packages, you can run the following command:
+
+```
+npm install @moonbeam-network/xcm-sdk @moonbeam-network/xcm-config
+```
+
+!!! note
+    There is a [known issue](https://github.com/polkadot-js/api/issues/4315){target=_blank} when using the Moonbeam XCM packages alongside Polkadot.js that will cause warnings to appear in the console.
```

Comment:
Maybe we can say that the issue appears only in JavaScript.

Reply:
I've added that in my commit :)